exts/cesium.omniverse/cesium/omniverse/ui/credits_viewport_frame.py

```diff
@@ -99,3 +101,9 @@ def _build_fn(self):
                         height=0,
                         clicked_fn=self._on_data_attribution_button_clicked,
                     )
+
+    def set_new_credits(self, credits):
+        self._new_credits = credits
+
+    def handle_event(self, event):
+        self._new_credits = event
```

Comment:
set_new_credits is unused. It's now gone. For a quick first pass I was using a setter instead of an event. I forgot to delete the setter.

My old IDE (Visual Studio) would flag unused methods (in C#/C++). I'd like to find a way to get VS Code to do that for Python.